### ISSUE TITLE
Persist conversation histories

### ISSUE THREAD

**Issue by nathanleclaire** [ollama/ollama#142]
It would be really fun and useful to have a `docker ps`-like persistence of each session 

```
SESSION_ID  MODEL   CREATED AT        LAST CHAT      SUMMARY
asdfasdfasd llama2  50 minutes ago    1 minute ago   Explorations with Waluigi
qwfdsqwefd  llama2  30 minutes ago    20 minutes ago Mario is Mean
```

This would also provide a basis for syncing them around. I'd really like to be able to continue a chat from one computer on another! 

Then eventually maybe even something like https://github.com/jmorganca/ollama/pull/69 could be expanded to all history or a subset of it.

**Comment by pdevine** [comment]
I love this. We've been working on the session stuff to make sure it's running correctly and I've been thinking about how the sessions could be broken up into other layers and you could potentially push them and have someone else continue a conversation. So many possibilities there.

**Comment by pdevine** [comment]
This is now implemented with #2063
